src/pkg/projects/windowsdesktop/src/windowsdesktop.depproj

```diff
@@ -1,4 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
This should have been fixed by https://github.com/dotnet/core-setup/commit/939f0a7116eee51f44cb4ea5396d781c2ab1c6ce: https://github.com/dotnet/core-setup/blob/412e8f8ebb1fb24e41940fed7f3eb99d7a2d3028/src/pkg/projects/windowsdesktop/Directory.Build.props#L21-L25

Do you know for sure if the patch is still necessary after that? (No harm adding a potentially redundant exclude if it would take a lot of effort to check, though.)

Reply:
It shouldn't be necessary anymore, thanks.